restaurant/src/assets/home.jsx

```diff
@@ -0,0 +1,27 @@
+import "./home.css";
+export default function Home() {
+  return (
+    <div className="home-page">
+      <div className="Homeheader">Bunnie Tea House</div>
+      <div className="Homeinfo">
```

Comment:
nit comment,

You have inconsistent naming.
- some have all lowercase `home-page`
- some have i don't know what you would call it `Homeinfo`

You should stick with one, such as `home-page` & `home-info` OR `homePage` & `homeInfo`